cardano-launcher/configuration/launcher/launcher-config.demo.yaml

```diff
@@ -22,7 +22,7 @@ nodeArgs:
 - --wallet-address
 - 127.0.0.1:0
 - --update-with-package
-statePath: ${XDG_DATA_HOME}/Daedalus/mainnet
+statePath: .
```

Comment:
This config file is used to test the launcher on our local machine, modifying the value should not matter.

Reply:
it may help to just make a custom launcher config for each testcase, rather then trying to use copies of the generated config